Create ux-writer-role.md (#202)
siemens/ix-docs: docs/guidelines/language/support-and-resources/ux-writer-role.md

---
sidebar_position: 3
sidebar_label: UX writer role
title: UX writer role
doc-type: 'banner'
component-tabs: ['']
no_single_tab: true
description: UX writers shape the language that bring our digital products to life. Through clear, consistent and accurate microcopy (the small, functional pieces of text within a digital interface that guide, support and reassure users), UX writers move users easily and confidently through complex workflows.
---

import React from "react";
import { IxIcon } from "@siemens/ix-react";
import { iconOpenExternal } from "@siemens/ix-icons/icons";

#

> ## Overview

Reviewer: Maybe it's good to clearly separate them from visual/UX designers, or why designers aren't writers (usually "it's all UX, so what's the deal" 😂)

Reviewer 2: want to limit scope here only to UX writer- we arent responsible for the other roles - and are waiting for Siemens to finish their work - we want to publish this before as there is no date for their role definitions = except ours is now finished.

UX writers are part of the design team and coordinate with multiple stakeholders.

## UX writing is designing

UX writing is an integral part of design, not an afterthought. Writers participate throughout the design process, from early user experience (UX) research to final implementation, shaping how users understand and interact with the product.

Writers collaborate closely with all members of the product team: UX/UI designers, UX researchers, developers, architects, product owners and business owners, etc. to ensure wording supports interactions and functionality.

UX research provides the foundation: understanding user terminology, mental models and pain points ensures language matches expectations. Through ongoing iteration, validated by usability testing, analytics and qualitative feedback, UX writers continuously refine the effectiveness of their wording.

## UX writing scope and collaboration

As part of the design team, UX writers partner with other specialized teams for content. This division of responsibilities ensures each content type receives specialized expertise while maintaining consistency across the user experience.

> ### Marketing and brand teams

Reviewer: Since all sections here are just one title + a short sentence, consider having a simple bullet list e.g.:

- **Marketing and brand teams** handle promotional language, persuasive messaging and brand storytelling that aims to engage rather than guide users through tasks.
- **Technical writing teams** own comprehensive user documentation, detailed tutorials, help articles and extended product explanations.

Author: according to accessibility issues we want to keep our fomatting

Reviewer 2: not using this

Handle promotional language, persuasive messaging and brand storytelling that aims to engage rather than guide users through tasks.

### Technical writing teams

Own comprehensive user documentation, detailed tutorials, help articles and extended product explanations.

### Legal teams

Are responsible for disclaimers, terms and conditions, privacy policies and all legally binding language.

### Product strategy

Manage narrative content, thought leadership and storytelling that doesn't directly support in-product user actions.

### Accessibility teams

Update team on new accessibility standards and norms, and test UX writing guidelines for accessibility.

### Development teams

Support writers to understand the technical processes behind the software to “translate” this to suitable and voice-aligned UI wording. They also support the accurate implementation of terminology and microcopy.

> ## Design systems

Reviewer: Since this document is shown in context of iX it would be nice it sounds like it's a part of it, not just attached. Not sure if an explanation about the contents of a design system is placed right here, in a sub-sub-...-level of the design systems docs.

> Design systems provide components, shared patterns and guidelines for consistent product language. They are used when writing microcopy, validate terminology before implementation and apply established patterns for consistency. UX writing patterns, templates and examples of best practice are shared here for teams to adopt and integrate into their product.

Reviewer 2: ```suggestion
As design systems, such as Industrial Experience, provide components, shared patterns and guidelines for consistent product language, UX writers use design systems to write microcopy and validate terminology before implementation. In addition, design systems can be used by UX writers to apply established patterns for consistency. UX writing patterns, templates and examples of best practice are shared here for teams to adopt and integrate into their product.
```

### What’s included

> * Voice and tone framework

Reviewer: as already discussed, please use minus for bullet points instead of asterisks :)

* Component guidelines (e.g. menus, dialogs, buttons, warnings, notifications)
* Standardized terminology and approved terms
* Writing guidelines and templates for components 
* Common writing issues reference, such as false friends, common spelling issues, etc. 

> ## UX writing responsibilities

Reviewer: this section talks about like everybody's main focus is contributing ux writing guidelines and owns them which is not true.

I assume it's rather that there's ux writers defining these guideliens on the ix site that are followed by ux writers that define microcopy in their projects.

Maybe it actually doesnt matter which topic the project has - if it is an app or a design system. Maybe it can be simply generic e.g. "contribute and support writing, and approve terminology changes"

Reviewer 2: made changes

### Ownership

> Senior UX writers and senior UX writing teams contribute to the design system, approve terminology changes and support the writing of component guidelines. These teams own the UX writing guidelines and the associated terminology databases.

Reviewer 2: ```suggestion
Senior UX writers and senior UX writing teams contribute to the design system with UX writing rules and templates and approve terminology changes. These teams own the UX writing guidelines and the associated terminology databases.
```

### Onboarding and training

New UX writers should complete an onboarding program covering the UX writing guidelines, design system and review processes. Ongoing training should also include workshops on best practices. All UX writers should have access to self-paced learning resources.

## UX writing in agile projects

Integrate UX writing throughout all agile project phases to allow product language to evolve with the design.

* Sprint planning: Identify content needs and terminology early in ideation.
* Daily standups: Stay aligned on interaction changes that affect microcopy.
* Refinement: Collaborate with designers and product owners on wording decisions.
* Development: Ensure accurate implementation and consistency.
* Retrospectives: Refine the writing process based on team feedback.

Continuous collaboration improves quality and speed, prevents terminology gaps, reduces rework and improves clarity throughout the product.

## Related

* [UX writing principles](./UX-writing-principles.md)

> * UX writing support and resources (coming soon)

Reviewer: aren't those markdowns already there? 🤔

Author: in another branch
